### ISSUE TITLE
Cannot queue more files to load with Upload when another file is still processing

### ISSUE THREAD

**Issue by martenson** [galaxyproject/galaxy#6271]
When this occurs, you cannot navigate away from the history or load more data -- just have to wait it out -- seems problematic,

How to reproduce:

Upload a file that takes a bit longer to process. Note that while that is processing (in upload tool), the buttons to start another upload are greyed out (not usable). As soon as the first file finishes queueing, then the buttons are active again.

<img width="890" alt="screen shot 2018-05-17 at 9 01 44 am" src="https://user-images.githubusercontent.com/8999735/40189657-d85e8878-59b1-11e8-9791-1766bb423e3d.png">

<img width="892" alt="screen shot 2018-05-17 at 9 03 05 am" src="https://user-images.githubusercontent.com/8999735/40189659-d8794140-59b1-11e8-9b9a-f3264eace0ed.png">

transferred from https://github.com/galaxyproject/usegalaxy-playbook/issues/126

**Comment by shiltemann** [comment]
Yeah, this is biting one of my researchers at the moment, I'll briefly explain her use case:

She has a set of folders with files, each folder needs to eventually go into its own collection, data is structured like:

```
run01
   | -- file1.fastq
   | -- file2.fastq
   | -- ..
   | -- fileX.fastq
run02
   | -- file1.fastq
   | -- file2.fastq
[..]
runX
   | -- files
```

The filenames don't show which run the files came from, so she also can't just start the upload for all at once and make the collections later.

The data is also large, so right now she has to start a collection upload, and wait til it's finished (often a long time since data is big and her internet is slow), and only then can queue the next upload job, so this can take her days in total if she has many runs.

It would be nice if she could queue multiple collection uploads at once.

Or if collections are harder, just being able to queue some regular uploads,  switch histories, queue some more, etc. to keep the runs separate would be ok too.
